manifests/dispatch_router.pp

```diff
@@ -1,81 +1,8 @@
 # @summary Install and configure Qpid Dispatch Router
```

Comment:
This is no longer correct.

Reply:
I could change this to 'Uninstall Qpid Dispatch Router' but this comment got me thinking that the class could just be removed and what little content remains could be moved into init.pp